sysutils/scp-backup/src/opnsense/mvc/app/controllers/OPNsense/ScpBackup/forms/general.xml

```diff
@@ -30,10 +30,22 @@
     <help>Set the remote directory to backup the config file to.</help>
   </field>
   <field>
-    <id>general.publickey</id>
-    <label>Local Root's Public Key</label>
+    <id>general.privatekey</id>
+    <label>SSH Private Key</label>
     <type>textbox</type>
-    <help>The public key of the local root user. This public key must be added to the remote user's authorized_keys - otherwise backups will fail.</help>
-    <readonly>true</readonly>
+    <help>The private key of the user. The paired public key must be added to the remote user's authorized_keys otherwise backups will fail.</help>
+    <height>10</height>
+  </field>
+  <field>
+    <id>general.stricthostkeycheck</id>
+    <label>Enable StrictHostKeyCheck</label>
+    <type>checkbox</type>
+    <help>If enabled, SSH will never automatically add host keys. The host key value must be provided below.</help>
+  </field>
+  <field>
+    <id>general.hostkey</id>
```

Comment:
remove

Reply:
Respectfully, I disagree. This is the point, not to have the user have to shell into the running OPNsense install to manually configure SSH information. It should all be done via the GUI.

Reply:
If you can tell me where the GUI is missing: https://github.com/opnsense/core/pull/2064/files
![screenshot_20180109_175506](https://user-images.githubusercontent.com/3713914/34732776-7e7ca894-f566-11e7-9eb0-2bb568e17282.png)

Reply:
BTW: key pair form looks like that:
![screenshot_20180109_175950](https://user-images.githubusercontent.com/3713914/34732891-e4316af8-f566-11e7-9721-101340361a74.png)